### DISCUSSION TITLE
request-reply特性(rpc)，客户端支持其他语言吗

### DISCUSSION THREAD

**Discussion by ddfnddfn2** [apache/rocketmq#7278] [Q&A]
没有找到其他语言的案例，客户端支持C#、C++吗

**Comment by mxsm** [comment]
@ddfnddfn2 你可看一下这个https://github.com/apache/rocketmq-clients，
基于proxy的实现。其他语言的实现都是基于proxy来实现的。没有单独的直接实现客户端直连MQ

**Comment by RongtongJin** [comment]
> 没有找到其他语言的案例，客户端支持C#、C++吗

暂时不支持。
